content/hi/docs/reference/glossary/pod-disruption.md

```diff
@@ -0,0 +1,22 @@
+---
+title: पोड डिसृपशन (Pod Disruption)
+id: pod-disruption
+date: 2021-05-12
+full_link: /docs/concepts/workloads/pods/disruptions/
+short_description: >
+  एक प्रक्रिया जिससे नोड्स के ऊपर जो पॉड्स है वो अपनी इच्छा से या फिर उसके विपरीत समाप्त हो सके। 
+  
+aka:
+related:
+ - pod
+ - container
+ tags:
+  - operation
+---
+
+[पोड डिसृपशन](/docs/concepts/workloads/pods/disruptions/) एक प्रक्रिया है जिससे नोड्स के ऊपर जो पॉड्स है वो अपनी इच्छा से या फिर उसके विपरीत समाप्त हो सके। 
+
+<!--more-->
+
+इच्छा से किये गए डिसृपशन एप्लीकेशन मालिक या फिर क्लस्टर प्रशासक जानबूझकर चालु करते है।
+बिना इच्छा से किये गए डिसृपशन अनजाने में होते है और वो ट्रिगर उन वजह से होते है जिन्हे टाला नहीं जा सकता जैसे की नोड्स के पास संसाधन ख़तम हो जाना या फिर अनजाने में मिट जाने से। 
```

Comment:
I'd suggest using google translate to start with. Many words such as trigger and जानबूझकर can be replaced with better choice of words.

Reply:
Done.